shm/test.cpp

```diff
@@ -0,0 +1,41 @@
+#include "cargo.hpp"
+#include "coordinates.hpp"
+#include "island.hpp"
+#include "map.hpp"
+#include "player.hpp"
+#include "ship.hpp"
+
+#include <vector>
+#include "gtest/gtest.h"
```

Comment:
Why vector(standard library) and gtest include(header includes) are in the same paragraph.

Reply:
Whole test.cpp was not used, I deleted it, but thank you for comment